test/snippet/utility/bloom_filter/bloom_filter_count.cpp

```diff
@@ -0,0 +1,29 @@
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+#include <seqan3/core/debug_stream.hpp>
+#include <seqan3/search/views/kmer_hash.hpp>
+#include <seqan3/utility/bloom_filter/bloom_filter.hpp>
+
+
+int main()
+{
+    using namespace seqan3::literals;
+
+    seqan3::bloom_filter bf{seqan3::bin_size{8192u},
+                            seqan3::hash_function_count{2u}};
+
+    auto const sequence1 = "ACTGACTGACTGATC"_dna4;
+    auto const sequence2 = "GTGACTGACTGACTCG"_dna4;
+    auto const sequence3 = "AAAAAAACGATCGACA"_dna4;
+    auto hash_adaptor = seqan3::views::kmer_hash(seqan3::ungapped{5u});
```

Comment:
```suggestion
    auto 5mers = seqan3::views::kmer_hash(seqan3::ungapped{5u});
```

Reply:
I will change this to
```c++
auto kmers = seqan3::views::kmer_hash(seqan3::ungapped{5u});
```
to not start the variable name with a number.